docs/commands.md

```diff
@@ -27,6 +27,7 @@ The following sections describe the supported commands.
 | [`RG.TRIGGER`](#rgtrigger) | Triggers execution of registration |
 | [`RG.TRIGGERONKEY`](#rgtriggeronkey) | Triggers execution of registration on a given key |
 | [`RG.UNREGISTER`](#rgunregister) | Removes registration |
+| [`RG.CLEARREGISTRATIONSSTATS`](#rgunregister) | Clears stats from all registrations |
```

Comment:
The link to rgrunregister should be different, no?

Reply:
Thanks, fixed.